easybuild/easyconfigs/c/CMSeq/CMSeq-1.0.3-foss-2020a-Python-3.8.2.eb

```diff
@@ -0,0 +1,46 @@
+# Contribution by
+# DeepThought, Flinders University
+# R.QIAO <rob.qiao@flinders.edu.au>
+
+easyblock = 'PythonPackage'
+
+name = 'CMSeq'
+version = '1.0.3'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/SegataLab/cmseq/'
+
+description = """
+CMSeq is a set of commands to provide an interface to .bam files for coverage and sequence consensus.
+"""
+
+toolchain = {'name': 'foss', 'version': '2020a'}
+
+sources = ['%(name)s-%(version)s.tar.gz']
+checksums = ['c354b89bc4fdbb24fa1ffa063febdb96f2bbd8837af399887b300c9be98bda2e']
+
+dependencies = [
+    ('Python', '3.8.2'),
+    ('SciPy-bundle', '2020.03', versionsuffix),
+    ('Biopython', '1.78', versionsuffix),
+    ('bcbio-gff', '0.6.6', versionsuffix),
+    ('biom-format', '2.1.10', versionsuffix),
+    ('Pysam', '0.16.0.1'),
+    ('SAMtools', '1.12', '', ('GCC', '9.3.0')),
```

Comment:
@manifestoso please check whether you can use `SAMtools` `1.10` - see the unit test error for details.

Reply:
@verdurin fixed, thx.